include/SFML/Audio/SoundSource.hpp

```diff
@@ -21,8 +21,8 @@
 // 3. This notice may not be removed or altered from any source distribution.
 //
 ////////////////////////////////////////////////////////////
-
 #pragma once
+// NOLINTBEGIN(readability-make-member-function-const)
```

Comment:
Can this be moved down such that it only wraps the class and not also all those headers?